modules/gapi/src/streaming/onevpl/accelerators/accel_policy_va_api.cpp

```diff
@@ -0,0 +1,152 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
```

Comment:
2022

Reply:
please check the actual version of this functionality https://github.com/opencv/opencv/pull/21883/